### PR TITLE
Apidoc 111

### PR DESCRIPTION
**Pull request by NickCW99**
## Description

See the [review app](https://nexmo-developer-pr-3906.herokuapp.com/documentation).

### CODE REVIEW

**Comment by NickCW99** [comment]
Created Meetings API documentation as described in this outline: https://github.com/avital-vonage/meetings-api-docs/blob/master/docs.md
Ticket: https://jira.vonage.com/browse/APIDOC-111
Waiting for Meetings API Ref links to be setup

File: _documentation/en/meetings/overview.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+navigation_weight: 1
+---
+
+# Meetings API Overview
+
+The Meetings API allows you to integrate real-time, high-quality interactive video meetings into your web apps.
+
+## Contents
+
+* [Requirements](#requirements). What you will need to get started.
```
Comment (niati) [inline _documentation/en/meetings/overview.md:15]: Some of the cross-ref links here are not working. Also, remove the periods (.) after the x-ref links.

File: _documentation/en/meetings/code-snippets/create-instant-room.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+How to setup an Instant (default) room using the Meetings API.
+
+## Prerequisites
+
+* **Vonage Developer Account** If you don’t have a Vonage account yet, you can get one  here: [Vonage Developers Account](https://dashboard.nexmo.com/sign-up).
+
+* **Meetings API Activation** To activate the Meetings API, you must register. Please send an email request to the [Meetings API Team](mailto:meetings-api@vonage.com).
+
+* **API Key and Secret** Once you’re logged in, you'll find your API Key and Secret in your dashboard under "Getting Started".
```
Comment (niati) [inline _documentation/en/meetings/code-snippets/create-instant-room.md:17]: Can we include a hyperlink to the Dashboard menu to which we're referring? Is this it? https://dashboard.nexmo.com/getting-started/meetings-api

References to items on the UI do not go in quotes - i.e. "Getting Started" - you should apply bold font to any UI label. In this case, however, I would just reword "Once you're logged in to the Dashboard, you'll find your API Key and Secret on the Meetings API menu." Hyperlink to the Meetings API menu text to the actual dashboard page.

Also, when I land on the "Try the Meetings API" page, I have to Request Access to get started. Should that be included as a step here? That is probably a question for @avital-vonage

File: _documentation/en/meetings/code-snippets/create-instant-room.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+description: Setting up an Instant Meeting Room
+---
+
+# Setting up an Instant Meeting Room
+
+How to setup an Instant (default) room using the Meetings API.
+
+## Prerequisites
+
+* **Vonage Developer Account** If you don’t have a Vonage account yet, you can get one  here: [Vonage Developers Account](https://dashboard.nexmo.com/sign-up).
```
[the remark this answers is not in the document]
Comment (niati) [reply _documentation/en/meetings/code-snippets/create-instant-room.md:13]: Include a colon after each bullet point item in bold font - i.e. **Vonage Developer Account:**

File: _documentation/en/meetings/overview.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  * **Chat**: space for sending written messages that are visible to all attendees in the room.
+  * **Record**: you can record a meeting either manually during the meeting or automatically when sending a request.
+  * **Room Management**: you can delete, update or retrieve information about rooms.
+  * **Callbacks**: allow you to receiver information about a session.
+
+## Room Types
+There are two room types:
+
+* **Instant Room**:
+  * is created for a meeting happening now.
```
Comment (niati) [inline _documentation/en/meetings/overview.md:57]: Remove 'is' from these bullet points.
Make sure the first word in each bullet point is capitalized.

File: _documentation/en/meetings/code-snippets/create-instant-room.md (hunk 2 above)
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/create-instant-room.md:17]: Yes, I agree, probably similar in long term room. I'll change them both.

File: _documentation/en/meetings/code-snippets/create-instant-room.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/create-instant-room.md:13]: Done

File: _documentation/en/meetings/overview.md (hunk 4 above)
Comment (NickCW99) [reply _documentation/en/meetings/overview.md:57]: Done. Also line 33 create long term room long term should be long_term

File: _documentation/en/meetings/overview.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+## Contents
+
+* [Requirements](#requirements). What you will need to get started.
+* [Terminology](#Terminology). Key terms and definitions for the Meetings API.
+* [Room Types](Room-Types). Defines the types of rooms available
+* [Code Snippets](#code-snippets). Code and instructions for using the Meetings API.
+* [Reference](#reference). Further information about the Meetings API.
+
+## Requirements
+
```
[the remark this answers is not in the document]
Comment (NickCW99) [reply _documentation/en/meetings/overview.md:22]: Done

File: _documentation/en/meetings/overview.md (hunk 1 above)
Comment (NickCW99) [reply _documentation/en/meetings/overview.md:15]: Done, will push them through with your other changes

File: _documentation/en/meetings/overview.md (hunk 4 above)
Comment (NickCW99) [reply _documentation/en/meetings/overview.md:57]: Line 65, missing an end period.

File: _documentation/en/meetings/overview.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+# Meetings API Overview
+
+The Meetings API allows you to integrate real-time, high-quality interactive video meetings into your web apps.
+
+## Contents
+
+* [Requirements](#requirements): What you will need to get started.
+* [Terminology](#terminology): Key terms and definitions for the Meetings API.
+* [Room Types](#room-types): Defines the types of rooms available
```
Comment (avital-vonage) [inline _documentation/en/meetings/overview.md:17]: end of sentence periods are not consistent. My preference is without, considering these aren't full sentences.

File: _documentation/en/meetings/code-snippets/create-instant-room.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+## Prerequisites
+
+* **Vonage Developer Account**: If you do not already have one, sign-up for a free account on the [Vonage Developers Account](https://dashboard.nexmo.com/sign-up).
+
+* **Meetings API Activation**: To activate the Meetings API, you must register. Please send an email request to the [Meetings API Team](mailto:meetings-api@vonage.com).
+
+* **API Key and Secret**: Once you’re logged in to the [Vonage API Dashboard](https://dashboard.nexmo.com), you'll find your API Key and Secret on the Meetings API menu.
+
+## Setup POST Request
```
Comment (avital-vonage) [inline _documentation/en/meetings/code-snippets/create-instant-room.md:19]: Set up is two words

File: _documentation/en/meetings/code-snippets/create-instant-room.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
+---
+title: Create an Instant Room
+navigation_weight: 0
+description: Setting up an Instant Meeting Room
+---
+
+# Setting up an Instant Meeting Room
+
+How to setup an Instant (default) room using the Meetings API.
```
Comment (avital-vonage) [inline _documentation/en/meetings/code-snippets/create-instant-room.md:9]: set up should be two rooms

File: _documentation/en/meetings/code-snippets/create-instant-room.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+* **API Key and Secret**: Once you’re logged in to the [Vonage API Dashboard](https://dashboard.nexmo.com), you'll find your API Key and Secret on the Meetings API menu.
+
+## Setup POST Request
+
+**POST Endpoint**: The (POST) endpoint when creating a meeting room is: ``https://api.vonage.com/beta/meetings/rooms``.
+
+**Required Headers**: You need to add the ``Content-Type`` to your headers: ``Content-Type: application/json``.
+
+**Authorization**: Basic [Authentication](/concepts/guides/authentication) is enabled with your `VONAGE_API_KEY` and `VONAGE_API_SECRET` from your account.
+
```
Comment (avital-vonage) [inline _documentation/en/meetings/code-snippets/create-instant-room.md:26]: saying "is enabled" is inaccurate. It's an encoded string you need to create with your api key +secret, and i think we should be explicit about that. no need to send the developer searching for the answer

File: _documentation/en/meetings/code-snippets/create-instant-room.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+## Body Content
+
+You need to provide the following:
```
Comment (avital-vonage) [inline _documentation/en/meetings/code-snippets/create-instant-room.md:29]: You "need" to provide the following is misleading when some params are required and some aren't. We should remove this sentence entirely or reword the sentence.

File: _documentation/en/meetings/code-snippets/create-instant-room.md [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+
+* **Vonage Developer Account**: If you do not already have one, sign-up for a free account on the [Vonage Developers Account](https://dashboard.nexmo.com/sign-up).
+
+* **Meetings API Activation**: To activate the Meetings API, you must register. Please send an email request to the [Meetings API Team](mailto:meetings-api@vonage.com).
+
+* **API Key and Secret**: Once you’re logged in to the [Vonage API Dashboard](https://dashboard.nexmo.com), you'll find your API Key and Secret on the Meetings API menu.
+
+## Setup POST Request
+
+**POST Endpoint**: The (POST) endpoint when creating a meeting room is: ``https://api.vonage.com/beta/meetings/rooms``.
```
Comment (avital-vonage) [inline _documentation/en/meetings/code-snippets/create-instant-room.md:21]: This was my mistake. That won't work. Let's use `https://api-eu.vonage.com/beta/meetings/rooms`

Also, can we reword to "The endpoint for creating a meeting room is:"

File: _documentation/en/meetings/code-snippets/room-management.md [hunk 12]
```diff
@@ -0,0 +1,183 @@
...
+* [Callbacks](#callbacks).
+* [Recordings](#recordings).
+
+## Overview
+
+The Meetings API provides the following Room Management features.
+
+## Individual Room Retrieval
+
+Notice the ``ID`` received in the response. This is the ``ID`` of the room which will be used for room retrieval:
```
Comment (avital-vonage) [inline _documentation/en/meetings/code-snippets/room-management.md:25]: ...using a GET request.

File: _documentation/en/meetings/code-snippets/room-management.md [hunk 13]
```diff
@@ -0,0 +1,183 @@
...
+
+### Types of Callbacks
+
+* **Room Expired** ``room:expired``: A notification that a room is inactive. Sessions cannot be created for inactive rooms.
+* **Session Started** ``session:started``: A notification that a session has started.
+* **Session Ended** ``session:ended``: A notification that a session has finished.
+* **Recording Started** ``recording:started``: A notification about a recording beginning within a session.
+* **Recording Ended** ``recording:ended``: A notification about a recording being stopped within a session.
+* **Recording Uploaded** ``recording:uploaded``: A notification that a recording is ready to be downloaded.
+* **Participant Joined** ``participant-joined``: A notification about someone joining a session.
```
Comment (avital-vonage) [inline _documentation/en/meetings/code-snippets/room-management.md:89]: dash in participant-joint. should be a colon. my mistake, sorry

File: _documentation/en/meetings/code-snippets/room-management.md [hunk 14]
```diff
@@ -0,0 +1,183 @@
...
+* **Participant Left** ``session:participant:left``: A notification about someone leaving a session.
+
+### Example Payloads
+
+#### Session Started
+
+> A notification that a session has started.
```
Comment (avital-vonage) [inline _documentation/en/meetings/code-snippets/room-management.md:96]: Not sure it's accurate to put these in > format. I would just change these descriptions to regular text (for the following ones as well)

File: _documentation/en/meetings/code-snippets/room-management.md [hunk 15]
```diff
@@ -0,0 +1,183 @@
...
+    "started_at": "2021-11-02T15:55:35.000Z",
+    "ended_at": "2021-11-02T15:56:13.000Z",
+    "duration": 38,
+    "url": "https://prod-meetings-recordings.s3.amazonaws.com/46339892/17461b93-f793-48a0-9392-7d82de40432f/archive.mp4?..."
+}
+```
+
+## Recordings
+
+Recordings are associated with the session in which they occurred. To retrieve or manage recordings, you'll need the recording ID, which can be found in the callbacks.
```
Comment (avital-vonage) [inline _documentation/en/meetings/code-snippets/room-management.md:153]: backticks on ID :)

File: _documentation/en/meetings/code-snippets/room-management.md (hunk 12 above)
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/room-management.md:25]: Done

File: _documentation/en/meetings/code-snippets/room-management.md (hunk 13 above)
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/room-management.md:89]: Changed * **Participant Joined** to ``*participant:joint**``....

Is this correct, should it be **participant:joined**?

File: _documentation/en/meetings/overview.md (hunk 6 above)
Comment (NickCW99) [reply _documentation/en/meetings/overview.md:17]: Yes, these, we use periods when there is a leading sentence with colon. Done

File: _documentation/en/meetings/code-snippets/create-instant-room.md (hunk 7 above)
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/create-instant-room.md:19]: Yes, setup is a noun, set up is US verb, set-up is UK English verb. Changed to set up, both instance (line 9 as well).

File: _documentation/en/meetings/code-snippets/create-instant-room.md (hunk 8 above)
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/create-instant-room.md:9]: Changed to set up.

File: _documentation/en/meetings/code-snippets/create-instant-room.md (hunk 9 above)
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/create-instant-room.md:26]: I've checked to see what done elsewhere in the docs. Changed to 

`**Authorization**: Log into your [Vonage API Dashboard](https://dashboard.nexmo.com) to create your `VONAGE_API_KEY` and `VONAGE_API_SECRET`. You use the API key and secret for basic [Authentication](/concepts/guides/authentication).`

File: _documentation/en/meetings/code-snippets/create-instant-room.md (hunk 10 above)
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/create-instant-room.md:29]: I agree, only the display name is required. 

I've converted the list to a table, similar for long term meeting room. This means we can add extra fields and values. Comments very welcome. I nearly did this is a table in the first place:

image

File: _documentation/en/meetings/code-snippets/create-instant-room.md (hunk 11 above)
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/create-instant-room.md:21]: Done

File: _documentation/en/meetings/code-snippets/create-instant-room.md (hunk 9 above)
Comment (avital-vonage) [reply _documentation/en/meetings/code-snippets/create-instant-room.md:26]: No, you are not creating the api key and secret. 

**Authorization**: Log into your [Vonage API Dashboard](https://dashboard.nexmo.com) to retrieve your VONAGE_API_KEY and VONAGE_API_SECRET. You'll combine these to create a [basic Authentication](/concepts/guides/authentication) string.

File: _documentation/en/meetings/code-snippets/create-instant-room.md (hunk 10 above)
Comment (avital-vonage) [reply _documentation/en/meetings/code-snippets/create-instant-room.md:29]: Much better. love it.

File: _documentation/en/meetings/code-snippets/room-management.md (hunk 13 above)
Comment (avital-vonage) [reply _documentation/en/meetings/code-snippets/room-management.md:89]: yes. 
notice that it's joined and not joint :)

File: _documentation/en/meetings/code-snippets/room-management.md (hunk 15 above)
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/room-management.md:153]: Done

File: _documentation/en/meetings/code-snippets/create-instant-room.md (hunk 2 above)
Comment (avital-vonage) [reply _documentation/en/meetings/code-snippets/create-instant-room.md]: Sorry, just seeing this now. 

* **Meetings API Activation** To activate the Meetings API, you'll need to register your account. Please send an email request containing your API Key to the [Meetings API Team](mailto:meetings-api@vonage.com). 
You can also try it out in the [Vonage Dashboard](https://dashboard.nexmo.com/getting-started/meetings-api).

File: _documentation/en/meetings/code-snippets/room-management.md (hunk 14 above)
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/room-management.md:96]: There's a few of these, I'd like to find out what @niati (Nick Iati) thinks. Then we can target and define what a  note actually is.

File: _documentation/en/meetings/code-snippets/room-management.md (hunk 13 above)
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/room-management.md:89]: OK, just wondered.

File: _documentation/en/meetings/code-snippets/create-instant-room.md (hunk 8 above)
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/create-instant-room.md:9]: done, set up and set-up are US/UK verbs setup is a noun, as you correctly pointed out it is the wrong word, thanks.

File: _documentation/en/meetings/code-snippets/create-instant-room.md (hunk 9 above)
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/create-instant-room.md:26]: Done. I see what you mean, sorry,

File: _documentation/en/meetings/overview.md [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+
+* **Room**: the virtual space in which the video meeting takes place. See [Room Types](Room-Types) below.
+* **Participants**:
+  * **Owner**: usually the creator of the room, this person has special administration capabilities.
+  * **Guest**: people attending meeting. Guests have access to standard meeting features.
+* **Set up and Configuration**:
+  * **Request**: the code snippet you submit to set up a room.
+  * **Response**: the data that is returned from your request.
+  * **Meeting Room ID**: the ``ID`` is a session identifier which is returned in the response to a meeting request.
+  * **Session**: the time in which there are participants present in the room. Defined as from the first participant to join, until the last to leave.
```
Comment (avital-vonage) [inline _documentation/en/meetings/overview.md:46]: ...."Defined as all events that occur during this time, from when the first participant joins, until the last to leave."

File: _documentation/en/meetings/overview.md [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+* **Set up and Configuration**:
+  * **Request**: the code snippet you submit to set up a room.
+  * **Response**: the data that is returned from your request.
+  * **Meeting Room ID**: the ``ID`` is a session identifier which is returned in the response to a meeting request.
+  * **Session**: the time in which there are participants present in the room. Defined as from the first participant to join, until the last to leave.  
+  * **Guest URL**: meeting room URL used by guests.
+  * **Host URL**: meeting room URL with meeting administration capabilities used by the host.
+* **Features**:
+  * **Chat**: space for sending written messages that are visible to all attendees in the room.
+  * **Record**: you can record a meeting either manually during the meeting or automatically when sending a request.
```
Comment (avital-vonage) [inline _documentation/en/meetings/overview.md:51]: **Record**: you can record a meeting manually during the meeting, or automatically when a session is started.

File: _documentation/en/meetings/overview.md [hunk 18]
```diff
@@ -0,0 +1,78 @@
...
+  * **Room Management**: you can delete, update or retrieve information about rooms.
+  * **Callbacks**: allow you to receiver information about a session.
```
Comment (avital-vonage) [inline _documentation/en/meetings/overview.md:53]: **Callbacks**: allow you to receive information about a session.

(Typo on receiver)

Comment (NickCW99) [reply _documentation/en/meetings/overview.md:53]: Missed one, well caught!

**Comment by NickCW99** [comment]
1. Please can a curl dev check the changes we have made to standardise the curl code snippets. These are in

- Create an Instant Room
- Create a Long Term Room
- Meeting Room Management
- Callbacks

2. I've also added the api ref URL at the end of overview.md and the meetings api link in the left nav bar is working now.
3. Extra line in the code snippets list removed from overview.md#reference (at the end of the page)

**Comment by NickCW99** [comment]
Please can a curl dev check the changes we have made to standardise the curl code snippets. These are in

- Create an Instant Room
- Create a Long Term Room
- Meeting Room Management

I've also added the api ref URL at the end of overview.md and the meetings api link in the left nav bar is working now.
Extra line in the code snippets list removed from overview.md#reference (at the end of the page)

**Comment by NickCW99** [comment]
The Cross ref for the following link was incorrect. Not only did I not test it correctly, it was not caught by automated testing and reviews. One for the future :-) 

image

File: _documentation/en/meetings/code-snippets/create-instant-room.md [hunk 19]
```diff
@@ -0,0 +1,100 @@
...
+
+How to set up an Instant (default) room using the Meetings API.
+
+## Prerequisites
+
+* **Vonage Developer Account**: If you do not already have one, sign-up for a free account on the [Vonage Developers Account](https://dashboard.nexmo.com/sign-up).
+
+* **Meetings API Activation**: To activate the Meetings API, you must register. Please send an email request to the [Meetings API Team](mailto:meetings-api@vonage.com).
+
+* **API Key and Secret**: Once you’re logged in to the [Vonage API Dashboard](https://dashboard.nexmo.com), you'll find your API Key and Secret on the Meetings API menu.
```
[the remark this answers is not in the document]
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/create-instant-room.md:17]: Slight change to meet in-house standards.

File: _documentation/en/meetings/code-snippets/create-instant-room.md [hunk 20]
```diff
@@ -0,0 +1,100 @@
...
+
+## Set up POST Request
+
+**POST Endpoint**: The endpoint for creating a meeting room is: ``https://api-eu.vonage.com/beta/meetings/rooms``.
+
+**Required Headers**: You need to add the ``Content-Type`` to your headers: ``Content-Type: application/json``.
+
+**Authorization**: Log into your [Vonage API Dashboard](https://dashboard.nexmo.com) to retrieve your `VONAGE_API_KEY` and `VONAGE_API_SECRET`. You'll combine these to create a [Basic Authentication](/concepts/guides/authentication) string.
```
[the remark this answers is not in the document]
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/create-instant-room.md:25]: Slight changes to meet in-house standards.

File: _documentation/en/meetings/code-snippets/create-long-term-room.md [hunk 21]
```diff
@@ -0,0 +1,86 @@
...
+| | | If ``auto_record``=``true``, the session will be recorded.
+| | | If ``auto_record``=``false``, the session will not be recorded.
+
+## Request
+
+You can use the example code below to create a long term room which expires on October 21st 2022 and will be automatically recorded:
+
+``` curl
+curl -X POST 'https://api-eu.vonage.com/beta/meetings/rooms' \
+-H 'Authorization: basic YWFhMDEyOmFiYzEyMzQ1Njc4OQ==' \
```
[the remark this answers is not in the document]
Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/create-long-term-room.md:47]: FYI: Also added the two pieces of **Authorization** and **Application ID and Secret** content, so it is the same as instant room

File: _documentation/en/meetings/overview.md [hunk 22]
```diff
@@ -0,0 +1,79 @@
...
+* [Terminology](#terminology): Key terms and definitions for the Meetings API.
+* [Room Types](#room-types): Defines the types of rooms available.
+* [Code Snippets](#code-snippets): Code and instructions for using the Meetings API.
+* [Reference](#reference): Further information about the Meetings API.
+
+## Requirements
+
+**Vonage Developer Account**: If you don’t have a Vonage account yet, you can get one here: [Vonage Developers Account](https://dashboard.nexmo.com/sign-up).
+
+**Meetings API Activation**: To activate the Meetings API, you need to register with us. Please send an email request to the [Meetings API Team](mailto:meetings-api@vonage.com).
```
Comment (avital-vonage) [inline _documentation/en/meetings/overview.md:27]: with your API Key from the [Vonage API Dashboard](https://dashboard.nexmo.com).

File: _documentation/en/meetings/overview.md [hunk 23]
```diff
@@ -0,0 +1,79 @@
...
+
+**API Key and Secret**: Once you’re logged in, you'll find your API Key and Secret in your dashboard under "Getting Started".
```
Comment (avital-vonage) [inline _documentation/en/meetings/overview.md:29]: **Application ID and Private Key**: Once you’re logged in to the [Vonage API Dashboard](https://dashboard.nexmo.com), click on Applications and create a new Application. Click  `Generate public and private key` and record the private key. 

**JSON Web Token (JWT)**: Use [this generator](https://developer.vonage.com/jwt) to create a JWT using the Application ID and Private Token mentioned above. For further details about JWTs, please see [Vonage Authentication](/concepts/guides/authentication).

File: _documentation/en/meetings/overview.md [hunk 24]
```diff
@@ -0,0 +1,79 @@
...
+
+Key | Description
```
Comment (avital-vonage) [inline _documentation/en/meetings/overview.md:31]: i think we should remove this section as it's no longer relevant. what do you think?

**Review by avital-vonage — CHANGES_REQUESTED** [review]

File: _documentation/en/meetings/overview.md (hunk 22 above)
Comment (NickCW99) [reply _documentation/en/meetings/overview.md:27]: Done

File: _documentation/en/meetings/overview.md (hunk 23 above)
Comment (NickCW99) [reply _documentation/en/meetings/overview.md:29]: Done with slight changes

File: _documentation/en/meetings/overview.md (hunk 24 above)
Comment (NickCW99) [reply _documentation/en/meetings/overview.md:31]: Removed

File: _documentation/en/meetings/overview.md [hunk 25]
```diff
@@ -0,0 +1,76 @@
...
+* [Code Snippets](#code-snippets): Code and instructions for using the Meetings API.
+* [Reference](#reference): Further information about the Meetings API.
+
+## Requirements
+
+**Vonage Developer Account**: If you don’t have a Vonage account yet, you can get one here: [Vonage Developers Account](https://dashboard.nexmo.com/sign-up).
+
+**Meetings API Activation**: To activate the Meetings API, you need to register with us. Please send an email request, with your API key from the [Vonage API Dashboard](https://dashboard.nexmo.com), to the [Meetings API Team](mailto:meetings-api@vonage.com).
+
+**Application ID and Private Key**: Once you’re logged in to the [Vonage API Dashboard](https://dashboard.nexmo.com), click on Applications and create a new Application. Click Generate ``public and private key`` and record the private key.
```
Comment (avital-vonage) [inline _documentation/en/meetings/overview.md:29]: **Application ID and Private Key**: Once you’re logged in to the [Vonage API Dashboard](https://dashboard.nexmo.com), click on Applications and create a new Application. Generate a public and private key and record the private key.

**Review by avital-vonage — APPROVED** [review]

File: _documentation/en/meetings/code-snippets/room-management.md [hunk 26]
```diff
@@ -0,0 +1,115 @@
...
+-H 'Authorization: Basic YWFhMDEyOmFiYzEyMzQ1Njc4OQ==' \
+-H 'Content-Type: application/json'
+```
+
+> This operation will return a ``204`` status.
+
+## Expiration Update
+
+A room can be updated by using a PATCH action and the room ID. Changes can be for ``display_name``, ``metadata``, ``expires_at``, and ``recording_options``
+> The expiration date of a long term room can be updated by using a PATCH action and the room ID. The new date should be included in an object called ``update_details``.
```
Comment (avital-vonage) [inline _documentation/en/meetings/code-snippets/room-management.md:46]: Please replace line 45 with whats in line 46. We changed things entirely.

Comment (NickCW99) [reply _documentation/en/meetings/code-snippets/room-management.md:46]: So delete line 45

**Review by niati — APPROVED** [review]

**Pull request merged**